python_modules/libraries/dagster-azure/dagster_azure/adls2/resources.py

```diff
@@ -70,6 +70,64 @@ class ADLS2Resource(ADLS2BaseResource):
 
     Contains a client for both the Data Lake and Blob APIs, to work around the limitations
     of each.
+
+    Example usage:
+
+    1. Attach this resource to your Definitions to be used by assets and jobs.
```

Comment:
I would remove the numbering from these so they don't scan as sequential steps. (I think we do that elsewhere, but it's not ideal.) I'd also add a colon at the end: `Attach this resource to your Definitions to be used by assets and jobs:`